config/data_sync_list/common.json

```diff
@@ -57,6 +57,12 @@
             "SyncDirection": "Active2Passive",
             "SyncType": "Immediate",
             "DestinationPath": "/var/lib/phosphor-data-sync/bmc_data_bkp/"
+        },
+        {
+            "Path": "/var/lib/service-config-manager/",
+            "Description": "Persisted service enable/disable config data",
+            "SyncDirection": "Active2Passive",
```

Comment:
We recently discussed that the service manager needs to run on both BMCs, so this should also be treated as bidirectional with notifications. We just need to ensure that the file path remains unique if both BMCs can update it, to avoid race conditions.

@geissonator any comments?